parser.y

```diff
@@ -634,6 +634,7 @@ import (
 	now                   "NOW"
 	position              "POSITION"
 	recent                "RECENT"
+	s3                    "S3"
```

Comment:
`"S3"` should be listed in `NotKeywordToken:`.

consider also add `S3` to the `unreservedKws` array in the `TestSimple` test in `parser_test.go`.

Reply:
already added at line 5598 in https://github.com/pingcap/parser/pull/1144/commits/dc7f9600822180f35ccdcedba9e467ad164482f1